File: docs/Acceleration/color/rgb/blue_color_generator.js
class BlueColorGenerator extends RGBColorGenerator {

    randomColor() {
        colorMode(RGB, 255);
        let r = int(random(50, 100));
        let g = int(random(50, 100));
        let b = int(random(100, 255));
        return color(r, g, b);
    }
}

File: Collisions/src/color/RGB/BlueColorGenerator.java
package color.RGB;

import processing.core.PApplet;

public class BlueColorGenerator extends RGBColorGenerator {

    public BlueColorGenerator(PApplet p) {
        super(p);
    }

    public int getRandomColor() {
        p.colorMode(PApplet.RGB, 255);
        int r = (int)p.random(50, 100);
        int g = (int)p.random(50, 100);
        int b = (int)p.random(100, 255);
        return p.color(r, g, b);
    }

}
